crud/crud_api.go

```diff
@@ -16,7 +16,7 @@ type APICrudModel interface {
 	List() interface{}
 }
 
-func CrudCreator(m APICrudModel) map[string]func(c *gin.Context) {
+func Creator(m APICrudModel) map[string]func(c *gin.Context) {
```

Comment:
naming is hazy a bit and confusing and by updating services that using goxy we'll get conflict

Reply:
Naming is not confusing because this linter comments seems fair:

```
Error: crud/crud_api.go:19:6: func name will be used as crud.CrudCreator by other packages, and that stutters; consider calling this Creator (golint)
func CrudCreator(m APICrudModel) map[string]func(c *gin.Context) {
```

And yes this is a breaking change but we have started versioning before this merge request and this will not break the projects using the v0.0.0 tag.

Reply:
ok